src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -19,12 +20,21 @@
         :container-style style/empty-container-style}]
       (into [rn/view {:style style/my-accounts-container}]
             (map (fn [{:keys [color address] :as account}]
-                   [quo/account-item
-                    {:account-props (assoc account :customization-color color)
-                     :on-press      #(rf/dispatch [:wallet/select-send-address
-                                                   {:address   address
-                                                    :recipient account
-                                                    :stack-id  :screen/wallet.select-address}])}]))
+                   (let [network-short-names (map network-utils/network->short-name
+                                                  (:network-preferences-names account))
+                         prefix              (when (< (count network-short-names) 3)
+                                               (network-utils/short-names->network-preference-prefix
+                                                network-short-names))
+                         address             (str prefix address)]
+                     [quo/account-item
+                      {:account-props (assoc account
+                                             :customization-color color
+                                             :address             address
+                                             :full-address?       true)
+                       :on-press      #(rf/dispatch [:wallet/select-send-address
+                                                     {:address   address
+                                                      :recipient account
+                                                      :stack-id  :screen/wallet.select-address}])}])))
             other-accounts))))
```

Comment:
May I suggest a cosmetic change, prefer `for` instead of `map` in long functions producing hiccup, otherwise the dev has to jump down/up sometimes just to see which sequence is being passed on. I wouldn't use this style in non-hiccup code, but here it's nice because we almost always just iterate on collections, and `for` helps with the readability and nesting of code.